### ISSUE TITLE
Use different explosion sprite

### ISSUE THREAD

**Issue by tobbi** [SuperTux/data#23]
Hello, I have found a pretty good explosion sprite from here: http://opengameart.org/content/simple-explosion-bleeds-game-art

I wonder how these look in-game. Can anyone try that out?

I'd say that replacing the explosion sprite would also fix the "sprite locked to 90 degrees intervals"

**Comment by uafr** [comment]
Am 21. Juli 2016 03:56:48 MESZ, schrieb Tobias Markus notifications@github.com:

> Hello, I have found a pretty good explosion sprite from here:
> http://opengameart.org/content/simple-explosion-bleeds-game-art
> 
> I wonder how these look in-game. Can anyone try that out?

This seems to be CC-BY, not GPL. Is that a problem?

**Comment by tobbi** [comment]
I don't think so. We have been using CC-BY for a few other things as well. As long as we note the license in-game, it should be fine (aka, the credits or something).

**Comment by Hypernoot** [comment]
Looks good, just needs a tiny lighting animation.

**Comment by maxteufel** [comment]
> This seems to be CC-BY, not GPL. Is that a problem?

It's less restrictive, so shouldn't be a problem. GNU says this can be used for art, documentation, etc.: https://www.gnu.org/licenses/license-list.html. In fact, I'd prefer less restrictive licenses rather than all these cancer-like licenses that require re-releasing under the same license or even more.
We also have more restrictively licensed works in data, e.g. `halloween_1.ogg` under CC-BY-NC (not recommended by GNU; we should avoid this in any case for the future).

It certainly looks more realistic, haven't seen it in the game yet, though.

**Comment by brmbrmcar** [comment]
I don't like the sprites as much, but I like the different colours.

I don't want to turn this even more off-topic, but CC BY-NC and the GPL/CC BY-SA (whatever this section is under) are not compatible.

**Comment by tobbi** [comment]
I only see CC-BY there. Nothing about share-alike. And we have other things in our tree that are CC-BY.

**Comment by Karkus476** [comment]
@brmbrmcar Why did you react to your own post with thumbs up?

@tobbi I am not sure about the style, this needs to be considered in context. We could use the different colours for the different bomb colours, too.
